webui/react/src/index.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Router } from 'react-router-dom';
-import { CompatRouter } from 'react-router-dom-v5-compat';
+/**
+ * It's considered unstable until `react-router-dom` can detect
+ * history version mismatches when supplying your own history.
+ * https://reactrouter.com/en/v6.3.0/api#unstable_historyrouter
+ */
+import { unstable_HistoryRouter as HistoryRouter } from 'react-router-dom';
```

Comment:
This should be safe cuz the version of `history` in `react-router-dom` is the same as `history` we use